views/settings.go

```diff
@@ -0,0 +1,51 @@
+package views
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"github.com/dustin/go-humanize"
+	"github.com/labstack/echo/v4"
+	"github.com/ystv/web-auth/templates"
+	"github.com/ystv/web-auth/user"
+	"strings"
+	"time"
+)
+
+type (
+	SettingsTemplate struct {
+		User       user.User
+		UserID     int
+		LastLogin  string
+		ActivePage string
+		Gravatar   string
+	}
+)
+
+// SettingsFunc handles a request to the internal template
+func (v *Views) SettingsFunc(c echo.Context) error {
+	session, _ := v.cookie.Get(c.Request(), v.conf.SessionCookieName)
+
+	c1 := v.getData(session)
+	lastLogin := time.Now()
+	if c1.User.LastLogin.Valid {
+		lastLogin = c1.User.LastLogin.Time
+	}
+
+	var gravatar string
+
+	if c1.User.UseGravatar {
+		hash := md5.Sum([]byte(strings.ToLower(strings.TrimSpace("liam.burnand@bswdi.co.uk"))))
```

Comment:
Hard-coded

Reply:
Relic from a mistaken merge

Reply:
Still present - which commit is it removed in?

Reply:
I remember removing it multiple times but somehow it stays